### YamlMime:ManagedReference
items:
- uid: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs
  id: ServicesExceptionNotHandledEventArgs
  children:
  - System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.Exception
  - System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.WorkflowInstanceId
  langs:
  - csharp
  name: ServicesExceptionNotHandledEventArgs
  nameWithType: ServicesExceptionNotHandledEventArgs
  fullName: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs
  type: Class
  summary: "Adja meg az adatait a <xref href=&quot;System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled&quot;> </xref> esemény."
  remarks: "> [!NOTE] >  [!INCLUDE[DeprecatedContent](~/add/includes/ajax-current-ext-md.md)]Ha egy szolgáltatás, amely származik a <xref:System.Workflow.Runtime.Hosting.WorkflowRuntimeService>osztály ütközik kivétel, amelyek nem tudják kezelni, akkor meghívhatja a <xref:System.Workflow.Runtime.Hosting.WorkflowRuntimeService.RaiseServicesExceptionNotHandledEvent%2A>metódus állapítható meg, hogy a munkafolyamat futásidejű motor kell kiadni a <xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled?displayProperty=fullName>esemény.</xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled?displayProperty=fullName> </xref:System.Workflow.Runtime.Hosting.WorkflowRuntimeService.RaiseServicesExceptionNotHandledEvent%2A> </xref:System.Workflow.Runtime.Hosting.WorkflowRuntimeService> A szolgáltatás továbbítja a <xref:System.Guid>a munkafolyamat-példány és a <xref:System.Exception>társított ezt az eseményt annak <xref:System.Workflow.Runtime.Hosting.WorkflowRuntimeService.RaiseServicesExceptionNotHandledEvent%2A>.</xref:System.Workflow.Runtime.Hosting.WorkflowRuntimeService.RaiseServicesExceptionNotHandledEvent%2A> hívása közben</xref:System.Exception> </xref:System.Guid> A munkafolyamat futásidejű motor magában foglalja a ServicesExceptionNotHandledEventArgs és vált ki ezeket a paramétereket a <xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled?displayProperty=fullName>esemény.</xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled?displayProperty=fullName> A belül egy előfizető számára, hogy a <xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled?displayProperty=fullName>esemény, áttekintheti a <xref:System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.WorkflowInstanceId%2A>tulajdonság és a <xref:System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.Exception%2A>tulajdonság használatával beolvassa az információk is társítva vannak egy adott <xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled?displayProperty=fullName>esemény.</xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled?displayProperty=fullName> </xref:System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.Exception%2A> </xref:System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.WorkflowInstanceId%2A> </xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled?displayProperty=fullName>"
  syntax:
    content: >-
      [System.Obsolete("The System.Workflow.* types are deprecated.  Instead, please use the new types from System.Activities.*")]

      public sealed class ServicesExceptionNotHandledEventArgs : EventArgs
  inheritance:
  - System.EventArgs
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.Exception
  id: Exception
  parent: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs
  langs:
  - csharp
  name: Exception
  nameWithType: ServicesExceptionNotHandledEventArgs.Exception
  fullName: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.Exception
  type: Property
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Lekérdezi a kivétellel, hogy a szolgáltatás nem tudta kezeli."
  syntax:
    content: public Exception Exception { get; }
    return:
      type: System.Exception
      description: "A <xref:System.Exception>, amely jelzi, hogy a kivétellel, hogy a szolgáltatás nem tudta kezeli.</xref:System.Exception>"
  overload: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.Exception*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.WorkflowInstanceId
  id: WorkflowInstanceId
  parent: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs
  langs:
  - csharp
  name: WorkflowInstanceId
  nameWithType: ServicesExceptionNotHandledEventArgs.WorkflowInstanceId
  fullName: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.WorkflowInstanceId
  type: Property
  assemblies:
  - System.Workflow.Runtime
  namespace: System.Workflow.Runtime
  summary: "Lekérdezi a <xref:System.Guid>a nem kezelt kivétel társított munkafolyamat-példány.</xref:System.Guid>"
  remarks: "A WorkflowInstanceId értéke megegyezik a <xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName>társított munkafolyamat-példány a <xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled>esemény.</xref:System.Workflow.Runtime.WorkflowRuntime.ServicesExceptionNotHandled> </xref:System.Workflow.Runtime.WorkflowInstance.InstanceId%2A?displayProperty=fullName>"
  syntax:
    content: public Guid WorkflowInstanceId { get; }
    return:
      type: System.Guid
      description: "A <xref:System.Guid>a nem kezelt kivétel társított munkafolyamat-példány.</xref:System.Guid>"
  overload: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.WorkflowInstanceId*
  exceptions: []
  platform:
  - net462
references:
- uid: System.EventArgs
  isExternal: false
  name: System.EventArgs
- uid: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.Exception
  parent: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs
  isExternal: false
  name: Exception
  nameWithType: ServicesExceptionNotHandledEventArgs.Exception
  fullName: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.Exception
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.WorkflowInstanceId
  parent: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs
  isExternal: false
  name: WorkflowInstanceId
  nameWithType: ServicesExceptionNotHandledEventArgs.WorkflowInstanceId
  fullName: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.WorkflowInstanceId
- uid: System.Guid
  parent: System
  isExternal: true
  name: Guid
  nameWithType: Guid
  fullName: System.Guid
- uid: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.Exception*
  parent: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs
  isExternal: false
  name: Exception
  nameWithType: ServicesExceptionNotHandledEventArgs.Exception
- uid: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs.WorkflowInstanceId*
  parent: System.Workflow.Runtime.ServicesExceptionNotHandledEventArgs
  isExternal: false
  name: WorkflowInstanceId
  nameWithType: ServicesExceptionNotHandledEventArgs.WorkflowInstanceId
